armonik/outputs.tf

```
@@ -1,3 +1,27 @@
+locals {
+
+  armonik_cli_config = {
+    endpoint              = coalesce(local.ingress_grpc_url, local.control_plane_url)
+    certificate_authority = abspath(one(local_sensitive_file.ingress_ca[*].filename))
```

`abspath` fails if input is null.

```suggestion
    certificate_authority = try(abspath(local_sensitive_file.ingress_ca[0].filename), null)
```